app/assets/src/components/views/visualizations/VisualizationsView.jsx

```diff
@@ -6,19 +6,23 @@ import cx from "classnames";
 
 import { Table } from "~/components/visualizations/table";
 import { humanize } from "~/helpers/strings";
-import GlobeIcon from "~ui/icons/GlobeIcon";
-import LockIcon from "~ui/icons/LockIcon";
+import { openUrl } from "~utils/links";
 
 import cs from "./visualizations_view.scss";
+import HeatmapPublic from "~ui/icons/HeatmapPublic";
+import HeatmapPrivate from "~ui/icons/HeatmapPrivate";
+import PhyloTreePublic from "~ui/icons/PhyloTreePublic";
+import PhyloTreePrivate from "~ui/icons/PhyloTreePrivate";
 
+// See also ProjectsView which is very similar
```

Comment:
What is the purpose of this comment?

Reply:
Yes, the purpose is to point out that there may be some implicit coupling. That is, if something changes in `ProjectsView` something else may need to change in `VisView`. Not having been a part of the design discussions or most of the coding, I just wanted to make note and not a write a `TODO`.